web_server/tests/urls/test_author_urls.py

```diff
@@ -0,0 +1,53 @@
+from django.test import SimpleTestCase
+from django.urls import reverse, resolve
+from authors.views import view_list_of_available_authors_to_befriend, unfriend, update_author_profile, retrieve_author_profile, post_creation_and_retrival_to_curr_auth_user, retrieve_posts_of_author_id_visible_to_current_auth_user, friend_checking_and_retrieval_of_author_id, check_if_two_authors_are_friends, post_edit_and_delete, post_creation_page
+
+
+class TestAuthorUrls(SimpleTestCase):
+
+    def test_post_creation_and_retrieval_to_curr_auth_user(self):
+        url = reverse("add_or_get_post")
+        self.assertEqual(resolve(url).func, post_creation_and_retrival_to_curr_auth_user)
```

Comment:
Just a note - this is a great start to a test case, but should not be a testcase itself. We should be testing

1. Whether the API maps to a specific function (e.g. checking hardcoded api endpoint urls to see if they map to a function, and then checking the functionality of that function to see if it behaves like the endpoint expects)